### DISCUSSION TITLE
Table of Content

### DISCUSSION THREAD

**Discussion by abwildan** [mnjm/kayal#26] [General]
Hi!
It's me again.

I'm not really sure whether this is kayal theme issue or simply hugo issue, but I think I need to ask about it here..

I had 4 subtitle in my blog, but they only appeared 2 in the Table of Content (ToC), can you help me with this?
![2024-08-14_04-22](https://github.com/user-attachments/assets/26f14569-4588-4789-a4b5-fdd4a9bbd4c7)

**Comment by mnjm** [comment]
I'm using Hugo's Inbuilt ToC nothing fancy. By default it only lists level 2 and 3 heading. You can configure this in `config/_default/markup.toml`. https://gohugo.io/methods/page/tableofcontents/

**Reply by abwildan** [reply]
thankie
